tests/snappy_pipeline/workflows/test_workflows_gene_expression_quantification.py

```diff
@@ -26,6 +26,14 @@ def minimal_config():
           reference:
             path: /path/to/ref.fa
 
```

Comment:
Please stick to two space indentation for YAML.

Reply:
Fixed.